shell/browser/api/atom_api_app.cc

```diff
@@ -898,9 +898,14 @@ void App::SetPath(gin_helper::ErrorThrower thrower,
 
   bool succeed = false;
   int key = GetPathConstant(name);
-  if (key >= 0)
+  if (key >= 0) {
     succeed =
         base::PathService::OverrideAndCreateIfNeeded(key, path, true, false);
+    if (key == DIR_USER_DATA) {
+      succeed |= base::PathService::OverrideAndCreateIfNeeded(
+          chrome::DIR_USER_DATA, path, true, false);
```

Comment:
why are we creating `chrome::DIR_USER_DATA` ?

Reply:
Chrome's hunspell implementation uses it to determine the user data dir to download hunspell dictionaries to.  I just set it up to exactly mirror our user data preference.

Reply:
@deepak1556 That's what this does (makes the two values the same thing).  `chrome::DIR_USER_DATA` is `1002` afaics.  `electron::DIR_USER_DATA` is `11000`.

Reply:
Resolved in DM.  This will be part of the whole "extract spellchecker from upstream into //component"